BuildResidentialHPXML/measure.xml

```diff
@@ -7270,6 +7270,2154 @@
       <usage_type>resource</usage_type>
       <checksum>D07918C5</checksum>
     </file>
+    <file>
+      <filename>extra_files/base-mf.osm</filename>
+      <filetype>osm</filetype>
+      <usage_type>test</usage_type>
+      <checksum>817B1238</checksum>
+    </file>
+    <file>
+      <filename>extra_files/base-mf.xml</filename>
```

Comment:
Can you update the tests so they clean up these files when they're finished? We don't want 2000+ new lines in the measure.xml Use the `teardown` method like we do for other measure tests.

Reply:
The test file already includes a `teardown` method that (optionally) cleans up these files. I had disabled it during testing. I will remove the folder and update the measure.xml.